client/reader/following-manage/feed-search-results.jsx

```diff
@@ -25,7 +25,7 @@ const FollowingManageSearchFeedsResults = ( {
 	searchResultsCount,
 	query,
 } ) => {
-	const isEmpty = !! ( query && searchResults && searchResults.length === 0 );
+	const isEmpty = !! ( query && query.length > 0 && searchResults && searchResults.length === 0 );
```

Comment:
Yep - shall I add to another PR? Should stop a flash of 'no sites matched %s' when the page is loading.